src/Routes/Router.js

```diff
@@ -0,0 +1,70 @@
+
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import HomePage from "../Screens/EnterSection/HomePage";
+import LoginPage from "../Screens/EnterSection/LoginPage";
+import Error from "../Screens/Error/Error";
+import FeedPage from "../Screens/Home/FeedPage";
+import SignUp from "../Screens/EnterSection/SignUpPage";
+import ShoppingCart from "../Screens/EnterSection/ShoppingCart";
+import Restaurant from "../Screens/Restaurants/RestaurantPage";
+import AdressPage from "../Screens/EnterSection/AdressPage";
+import SearchPlaceHolderPage from "../Screens/Home/SearchPlaceHolderPage";
+import SearchResultsPage from "../Screens/Home/SearchResultsPage";
+import ProfilePage from "../Screens/ProfileUser/ProfilePage"
+import AddressEditPage from "../Screens/ProfileUser/AddressEditPage"
+import SignUpEditPage from "../Screens/ProfileUser/SignUpEditPage"
+import Confirmation from "../Screens/Restaurants/ConfirmationPage";
+
+
+export default function Router() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/">
+          <HomePage />
+        </Route>
```

Comment:
Não precisava colocar a página inicial como uma página home só com o logo da rappi. Ou se for fazer isso, coloque um botão pra levar para a página de Login. Pois do jeito que tá, o usuário cai numa página com o logo, mas sem nenhuma forma de navegar até a página de Login ou Signup.